fair_mango/metrics/metrics.py

```diff
@@ -549,17 +586,20 @@ def summary(self) -> dict:
                     else:
                         self.result[target][self.label1] = key[1]
                         self.result[target][self.label2] = key[0]
+
         return self.result
 
-    def rank(self, pr_to_unp: bool = True) -> dict:
-        self.pr_to_unp = pr_to_unp
-        if self.result is None:
-            self.summary()
+    def rank(self) -> dict:
         result: dict = {}
```

Comment:
be more ambitious about your typing

Reply:
Typing for the returned values is improved in many methods now.